source/includes/facts-versioning.rst

```diff
@@ -1,6 +1,6 @@
 .. start-rewind-desc
 
-*Optional* Directs |command| to operate only on the object version(s) that
+Directs |command| to operate only on the object version(s) that
```

Comment:
Remove the manually formatted `*Optional*` from rewind and versions so it can be done with `:optional:` in the place this content is included. 

Other includes have the same issue, but these two are the ones my new page uses.